From ca0c84e8e822cd9479c1548d14d8aa973cc56db5 Mon Sep 17 00:00:00 2001
From: Tom Keel <thomas.keel@intel.com>
Date: Tue, 18 Sep 2012 19:12:55 +0200
Subject: Fix the SDK build.

In the top-level Makefile, a couple of chmod commands were recently
added to a recipe, but the targets of the chmod commands may not
exist in all cases, e.g. when building the SDK. If a target of either
of the chmod command is not there, the chmod command fails, and the
build process does, too.

This change makes 'make' ignore the exit status from the chmods.

Change-Id: I1620f6e6f77dbdbdef78239197eda949ead28471
Signed-off-by: Tom Keel <thomas.keel@intel.com>
---
 core/Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index acc73bf..c33f607 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -807,9 +807,9 @@ $(BUILT_SYSTEMIMAGE): $(INSTALLED_MODULES_TARGET) \
 	$(hide) rm -rf $(TARGET_OUT)/lib/firmware
 	$(hide) mkdir -p $(TARGET_OUT)/lib/
 	$(if $(INSTALLED_MODULES_TARGET),$(hide) tar -xz -C $(TARGET_OUT)/lib/ -f $(INSTALLED_MODULES_TARGET))
-	$(hide) chmod -fR g-w $(TARGET_OUT)/lib/modules
+	$(hide) -chmod -fR g-w $(TARGET_OUT)/lib/modules
 	$(if $(INSTALLED_KERNELFW_TARGET),$(hide) tar -xz -C $(TARGET_OUT)/lib/ -f $(INSTALLED_KERNELFW_TARGET))
-	$(hide) chmod -fR g-w $(TARGET_OUT)/lib/firmware
+	$(hide) -chmod -fR g-w $(TARGET_OUT)/lib/firmware
 	$(call build-systemimage-target,$@)
 
 INSTALLED_SYSTEMIMAGE := $(PRODUCT_OUT)/system.img
-- 
1.7.4.1

